src/main/scala/intent/sbt/Runner.scala

```diff
@@ -35,8 +35,8 @@ object IntentFingerprint extends SubclassFingerprint with
  * A single run of a single test (controlled by SBT)
  */
 class SbtRunner(
-  val args: Array[String],
-  val remoteArgs: Array[String],
+  val args: Array[String | Null],
+  val remoteArgs: Array[String | Null],
```

Comment:
**Overridden java array field**
A Scala field overrides some Java array field. We need to use `Array[T | Null]` instead of `Array[T]` for the overriding to work.